frontend/src/components/Search.tsx

```diff
@@ -33,9 +38,14 @@ function Search(props: Props) {
     }
   };
 
-  const style: React.CSSProperties = {};
+  const buttonStyle: React.CSSProperties = {};
+  const searchBarStyle: React.CSSProperties = {};
```

Comment:
Create a file Search.module.scss and CSS rules for each style

```
.small {
   height: 37px;
}

.wide {
   width: 260px;
}
```

then refactor the if to apply the proper CSS style give the condition

```
import styles from "./Search.module.scss";
...

<button style={props.size === "small" ? styles.small : ""} />

searchClasses = []
if (props.size === "small") {
   searchClasses.push("small");
   if (props.wide) {
      searchClassess.push("wide");
   }
}
<input style={searchClasses.join(" ")} />
```

That way the style is always in a css/scss file and the logic remains in a jsx file